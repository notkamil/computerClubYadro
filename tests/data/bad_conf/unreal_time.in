3
99:99 23:60
10