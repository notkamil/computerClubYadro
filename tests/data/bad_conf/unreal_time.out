99:99 23:60